vuepress/docs/v7.0/tutorials/create/mfe/widget-configuration.md

```diff
@@ -223,14 +230,16 @@ npm run build
 - `my-widget-config/static/css`
```

Comment:
indent 3 spaces so these bullets can be indented, for smoother look. Same for the steps below